components/x-follow-button/src/FollowButton.jsx

```diff
@@ -0,0 +1,70 @@
+import { h } from '@financial-times/x-engine';
+import { withActions } from '@financial-times/x-interaction';
+
+import Button from './Button';
+
+const followButtonActions = withActions(() => ({
+	onSubmitAction(event) {
+		event.preventDefault();
```

Comment:
Where does the code live for the action?

Reply:
When I discussed this with MattB long time ago, he asked me to keep this as a placeholder, and as far as I remember his explanation was that "it will be used by app". But I don't really understand it

Reply:
ohhh. yes, i did say that. it's not necessary, as long as you can pass `onClick` and `onSubmit` into `FollowButton`, and document those.